src/main/java/io/nats/client/Options.java

```diff
@@ -544,6 +546,10 @@ public class Options {
     private final ExecutorService executor;
     private final ServerListProvider serverListProvider;
 
+    // TODO SFF What is this
+    private final List<java.util.function.Consumer<HttpRequest>> httpRequestInterceptors;
+    private final Proxy proxy;

```

Comment:
Proxy support was needed so that we could run code that connects through http proxy servers.